hazelcast-integration/springboot-session-analysis/client/src/main/resources/hazelcast-client.xml

```diff
@@ -1,7 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<hazelcast-client xsi:schemaLocation="http://www.hazelcast.com/schema/client-config hazelcast-client-config-3.10.xsd"
-           xmlns="http://www.hazelcast.com/schema/client-config"
-           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+<hazelcast-client xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
Kept at 3.12.6 because I didn't know how to upgrade both jet and spring boot to work with 4.0.